cineast-api/src/main/java/org/vitrivr/cineast/api/rest/handlers/actions/mediaobject/FindObjectPaginationGetHandler.java

```diff
@@ -0,0 +1,62 @@
+package org.vitrivr.cineast.api.rest.handlers.actions.mediaobject;
+
+import static org.vitrivr.cineast.api.util.APIConstants.LIMIT_NAME;
+import static org.vitrivr.cineast.api.util.APIConstants.SKIP_NAME;
+
+import io.javalin.http.Context;
+import io.javalin.plugin.openapi.dsl.OpenApiBuilder;
+import io.javalin.plugin.openapi.dsl.OpenApiDocumentation;
+import java.util.Map;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.vitrivr.cineast.api.messages.result.MediaObjectQueryResult;
+import org.vitrivr.cineast.api.rest.handlers.interfaces.GetRestHandler;
+import org.vitrivr.cineast.core.db.dao.reader.MediaObjectReader;
+import org.vitrivr.cineast.standalone.config.Config;
+
+public class FindObjectPaginationGetHandler implements GetRestHandler<MediaObjectQueryResult> {
+
+  public static final String ROUTE = "find/object/all/{" + SKIP_NAME + "}/{" + LIMIT_NAME + "}";
+
+  private static final Logger LOGGER = LogManager.getLogger(FindObjectPaginationGetHandler.class);
+
+  @Override
+  public MediaObjectQueryResult doGet(Context ctx) {
+    final Map<String, String> parameters = ctx.pathParamMap();
+
+    final var skipParam = parameters.get(SKIP_NAME);
+    final var skip = skipParam == null ? 0 : Integer.parseInt(skipParam);
+    final var limitParam = parameters.get(LIMIT_NAME);
+    final var limit = limitParam == null ? Integer.MAX_VALUE : Integer.parseInt(limitParam);
```

Comment:
If the supplied parameter cannot be parsed to an integer, a runtime exception will be thrown which will cause an unhelpful stacktrace to be returned to the caller. The parsing should be moved into the `try/catch` block, such that you can manually return a readable error message.

Reply:
The code now returns an empty object, which is consistent with the rest of the API.